bin/sugar-activity

```diff
@@ -89,19 +89,31 @@ def main():
 
     logger.start()
 
-    if 'SUGAR_BUNDLE_PATH' not in os.environ:
-        print 'SUGAR_BUNDLE_PATH is not defined in the environment.'
-        sys.exit(1)
-
-    if len(args) == 0:
-        print 'A python class must be specified as first argument.'
-        sys.exit(1)
+    os.environ['SUGAR_BUNDLE_PATH'] = os.path.abspath(os.curdir)
 
     bundle_path = os.environ['SUGAR_BUNDLE_PATH']
     sys.path.append(bundle_path)
 
     bundle = ActivityBundle(bundle_path)
 
+    if len(args) == 0:
+        if bundle.get_command().startswith('sugar-activity'):
+            print 'Guessing python class!'
+            command = bundle.get_command().split(" ")[1]
+        else:
+            print 'A python class must be specified as first argument ' + \
+                    'or else run it inside a python gtk3 activity directory.'
+            sys.exit(1)
+    else:
+        command = args[0]
+
+    if 'SUGAR_VERSION' not in os.environ:
```

Comment:
Mate, your launcher really should put sugar version in the environment.  Browse needs to use it, for example.